lua/obsidian/config.lua

```diff
@@ -207,13 +209,15 @@ end
 
 ---@class obsidian.config.FinderMappingOpts
 ---@field new string|?
+---@field insert_link string|?
 config.FinderMappingOpts = {}
 
 ---Get defaults.
 ---@return obsidian.config.FinderMappingOpts
 config.FinderMappingOpts.default = function()
   return {
     new = "<C-x>",
+    insert_link = "<C-i>",
```

Comment:
I would chose a different mapping. `<C-i>` is the same as `TAB` and can cause surprising behaviour. 

Maybe `C-l` for **link**?

Just my 2 cents.

Reply:
Thx for the suggestion. Replaced <C-i> with <C-l>.